op-node/cmd/batch_decoder/get_cb

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# 8607517
+# Fetched Ranges
+# --start 8605000 --end 8606882
+# --start 8606882 --end 8610000
+# --start 8607295 --end 8610000
+# 8607538
+L1_RPC=https://eth-goerli.g.alchemy.com/v2/OotEfuaXY_56fzUP87A-rkLeo1SWQeas go run . fetch --start 8607500 --end 8607600 --inbox 0x8453100000000000000000000000000000000000 --sender 0x2d679b567dB6187c0C8323Fa982CfB88B74dBcc7 
```

Comment:
This is a live alchemy key - you should remove this + re-roll the key otherwise people will abuse it.

Reply:
Disabled this key